src/styles/_usage-graph.scss

```diff
@@ -9,16 +9,28 @@
       }
 
       &__title {
+        flex-grow: 1;
         padding: 0;
       }
 
       &__actions {
+        display: flex;
         margin: inherit;
+        margin-left: auto;
+
+        @media (max-width: $pf-global--breakpoint--sm) {
+          margin-left: inherit;
+          padding-left: 0;
+        }
+
+        & > .curiosity-select {
+          display: inline-block;
```

Comment:
optional: looks like pf declares this as "inline-block", shouldn't need this line